content/docs/testing-farm.md

```diff
@@ -265,6 +265,28 @@ execute:
     how: tmt
 ```
 
+Another high-profile tool is [`rpminspect`](https://github.com/rpminspect/rpminspect).
```

Comment:
This feels vague. Would be nice to have a more concrete sentence about what this tool is for.

Reply:
I'll improve it. Unfortunately rpminspect's landing page doesn't provide a clear description :/ will come up with something

Reply:
updated now